packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -40,6 +43,34 @@ fn api_documentation() -> Vec<openapi::OpenApi> {
     ]
 }
 
+async fn read_from_store<'pool, 'id: 'pool, T, O, P, I>(
+    pool: &'pool P,
+    identifier: I,
+) -> Result<O, StatusCode>
+where
+    P: StorePool,
+    P::Store<'pool>: crud::Read<'id, I, T, Output = O>,
```

Comment:
I don't know how this could be simplified because we somehow want to express, that the `Store` acquired from a pool should be able to read `T` identified by `I`. As mentioned below, `O` could be removed, but I wasn't able to find out the bounds. I will revisit this later again, maybe I'm smarter then 🙃